doc/faq.rst

```diff
@@ -257,3 +257,20 @@ Deep learning requires a rich vocabulary to define an architecture and the
 use of GPUs for efficient computing. However, neither of these fit within
 the design constraints of scikit-learn. As a result, deep learning is
 currently out of scope for what scikit-learn seeks to achieve.
+
+
+Why is my pull request not getting any attention?
+-------------------------------------------------
+The scikit-learn review process takes a significant amount of time, and
+contributors should not be discouraged by a lack of activity or review on
+their pull request. We care a lot about getting things right
+the first time, as maintenance and later change comes at a high cost.
+We rarely release any "experimental" code, so all of our contributions
+will be subject to high use immediately and should be of the highest
+quality possible initially.
+
+Beyond that, scikit-learn is limited in its reviewing bandwidth; many of the
+reviewers and core developers are working on scikit-learn on their own time.
+If a review of your pull request comes slowly, it is likely because the
+reviewers are busy. Please do not feel demotivated and close your pull request
```

Comment:
I think I would remove the last sentence and maybe add just something asking for their understanding. Asking people not to feel demotivated is odd imho.